scripts/python/py_validator_pipeline/README.md

```diff
@@ -0,0 +1,95 @@
+# ETISS-GTS PIPELINE
```

Comment:
I would vote for moving the scripts and documentation to a third-party repository. It is a large number of files added to the ETISS source tree, which are only relevant for a subset of ETISS users.